### ISSUE TITLE
ReferenceError: Redactor is not defined

### ISSUE THREAD

**Issue by hybridinteractive** [craftcms/redactor#19]
I'm trying to add `alignment` to the redactor plugin.

According to the docs:

> just place their JS file (and CSS file if there is one) in `config/redactor/plugins/`. Both files should have the exact same name as the plugin ID that you will be typing into the plugins array in your Redactor config JSON.

In my `config/redactor/Simple.json` I have:

```
{
	"buttons": ["bold", "italic"],
	"plugins": ["source", "alignment"],
	"toolbarFixed": true
}
```

I downloaded script and added it here: `config/redactor/plugins/alignment.js` but then I get this: `Uncaught ReferenceError: Redactor is not defined  at Alignment.js:54`

I don't have to modify any `js` do I?

**Comment by jsunsawyer** [comment]
I saw this in Slack and tested it on my install. I'm not getting an error and the js file is loading, but I'm not seeing the alignment options on my redactor field.

**Comment by jsunsawyer** [comment]
Oops, yes. `Uncaught ReferenceError: Redactor is not defined at alignment.js:54`

**Comment by brandonkelly** [comment]
I'm guessing this issue is due to the fact that the documentation is all geared for Redactor 3.

We just released v2.0.0 which is powered by Redactor 3, so you should have better luck now.
